test/unit/utils/crc32.spec.js

```diff
@@ -17,15 +17,15 @@ describe("crc32", () => {
     };
     const resultOne = crc32(JSON.stringify(idsTohash));
     const resultTwo = crc32(JSON.stringify(idsTohash));
-    expect(typeof resultOne).toBe("string");
-    expect(resultOne).toBe("bc420422");
-    expect(resultTwo).toBe("bc420422");
+    expect(typeof resultOne).toBe("number");
```

Comment:
You don't have to change your code, but as an FYI, you can also do:

```
expect(resultOne).toEqual(jasmine.any(Number))
```

The error that shows up if it doesn't pass the assertion might be a little more friendly (I can't recall what it actually says), but other than that I think it's probably subjective which one is better.